### PR TITLE
Make the sample data generator script delete previous data by default

### LINKED ISSUE #34: [BUG] Avoid adding more sample data without removing old sample data with
**Describe the bug**

The sample data injector currently only injects data without checking if the data already exists. This shouldn't be the default behaviour.

***To Reproduce***

Run the `npm run fill-db` script multiple times and check the database

**Expected behavior**

The script should remove all previous data and add new data.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by snowteamer**
**Description**

This uses the `argparse` library to add a `-d` or `--drop` option to the sample data injection script.

This option allows to discard all existing product objects in the database.

**Related Issue**

Closes #34

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -10,24 +10,27 @@ const graphqlSchema = require('./schemas/graphql-schema');
...
 
-    app.use('/api', graphqlHTTP({ schema: graphqlSchema, graphiql: true }));
+  app.use('/api', graphqlHTTP({ schema: graphqlSchema, graphiql: true }));
 
-    app.get('/', (req, res) => {
-      res.send('Placeholder Text');
-    });
+  app.get('/', (req, res) => {
+    res.send('Placeholder Text');
+  });
```
Comment (Twaha-Rahman) [inline src/main.js:31]: Why did you refactor the code here? Your PR to refactor this code was already merged in a separate PR.

**Comment by snowteamer** [comment]
So we have to decide a name for that optional flag to keep the data. What about `--keep`?

**Comment by Twaha-Rahman** [comment]
> So we have to decide a name for that optional flag to keep the data. What about `--keep`?

That sounds good. Also keep a shorthand `-k`

**Comment by Twaha-Rahman** [comment]
Fix the merge conflict and we're good to go.

**Pull request merged**